redbeams/src/main/java/com/sequenceiq/redbeams/domain/stack/SslConfig.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.redbeams.domain.stack;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses.SslCertificateType;
+import com.sequenceiq.redbeams.repository.converter.SslCertificateTypeConverter;
+
+@Entity
+@Table
+public class SslConfig {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "sslconfig_generator")
+    @SequenceGenerator(name = "sslconfig_generator", sequenceName = "sslconfig_id_seq", allocationSize = 1)
+    private Long id;
+
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Column(name = "sslcertificate_value")
+    private Set<String> sslCertificates = new HashSet<>();
+
+    @Convert(converter = SslCertificateTypeConverter.class)
```

Comment:
In accordance with the DB schema change, if you agree, this should be extended with `@Column(nullable = false)`.